Frontend

Infitimely opened the pull request:
Added gallery and landing page. Added various files to support  
![Screen Shot 2021-03-24 at 9 47 21 PM](https://user-images.githubusercontent.com/73036505/112411257-9c73c200-8cea-11eb-8c89-00c8d18cf824.png)
![Screen Shot 2021-03-24 at 9 47 01 PM](https://user-images.githubusercontent.com/73036505/112411264-9e3d8580-8cea-11eb-97a9-b9def015684a.png)

AdmiralRustyShackleford committed:
Edited Login.js and merged Wireframe

Changed Login.js to generate a simple login page using Material UI and our color scheme. Not sure if it actually works lol

AdmiralRustyShackleford committed:
Update Login.js

Recreated basic login page

Infitimely committed:
added landing page

Infitimely committed:
added gallery page

Infitimely committed:
updated productConstants

amoghyatnatti committed:
Fixed container issue with pages

amoghyatnatti commented:
I would fully fledge out the formatting of the item listings and maybe change the navbar design in the landing page. The Collector name can blend in with the white too much.

Infitimely committed:
changed picture on landing page and redid header

AdmiralRustyShackleford committed:
Created forgot password page

I got that forgot password page set up so it can be plugged into the back end and added to as needed. Can be easily modified into a sign up/sign in page and probably an email support/feedback form as well (Just call it contact us I suppose)

AdmiralRustyShackleford committed:
Added profile page

I've put together a profile page from a template that looks close to what I'm imagining. I'm planning on replacing or editing a lot of the components to fit our needs better but this works for now

Infitimely committed:
added carousel to gallery page

Infitimely committed:
added footer

Infitimely committed:
deleted unnecessary file

Infitimely committed:
finished gallery page

amoghyatnatti committed:
Modified gallery formatting to have 4 listings per row

Infitimely commented:
Finalized gallery page. Michael has a lot of random stuff in there right now so I would probably wait before doing anything.
![Screen Shot 2021-03-31 at 10 04 32 PM](https://user-images.githubusercontent.com/73036505/113237940-6c3d9d80-926d-11eb-97d3-350b996ac4e2.png)

AdmiralRustyShackleford committed:
Pushed profile page

Forgot to push this with the assets last night, my bad

AdmiralRustyShackleford committed:
Merge branch 'frontend' of https://github.com/acm-projects/Collector into frontend

AdmiralRustyShackleford commented:
Added the profile page itself in. I'm gonna be modifying it and adding more in but this should be a good enough foundation. The assets I pushed last night are all just example photos used in the profile but they can be re used anywhere. Hope y'all like the spokesman I picked for the fake profile

amoghyatnatti committed:
Fixed all import errors with the profile page
